test/cluster.go

```diff
@@ -31,21 +32,23 @@ func CreateCluster(t *testing.T, db *gorm.DB) *repository.Cluster {
 }
 
 func NewCluster() *repository.Cluster {
+	random := uuid.NewV4()
 	return &repository.Cluster{
-		AppDNS:            uuid.NewV4().String(),
-		AuthClientID:      uuid.NewV4().String(),
-		AuthClientSecret:  uuid.NewV4().String(),
-		AuthDefaultScope:  uuid.NewV4().String(),
-		ConsoleURL:        uuid.NewV4().String(),
-		LoggingURL:        uuid.NewV4().String(),
-		MetricsURL:        uuid.NewV4().String(),
-		Name:              uuid.NewV4().String(),
-		SAToken:           uuid.NewV4().String(),
-		SAUsername:        uuid.NewV4().String(),
+		ClusterID:         random,
```

Comment:
Guys, you keep "optimizing" this code by using the same string for fields which are supposed to be unique :) The whole point here that these fields have different value. Not the same.

Reply:
I get your point, I just wanted to have a same UUID for all fields so I could much easily locate the cluster ID in the logs. Let me revert that now that I don't need it anymore

Reply:
done in ae521c6